src/Test/Utilities/CoreClr/TestExecutionLoadContext.cs

```diff
@@ -95,7 +95,7 @@ private Assembly LoadImageAsAssembly(ImmutableArray<byte> mainImage)
             }
         }
 
-        internal (int ExitCode, string Output) Execute(ImmutableArray<byte> mainImage, int expectedOutputLength)
+        internal (int ExitCode, string Output) Execute(ImmutableArray<byte> mainImage, int? expectedOutputLength)
```

Comment:
I don't see a reason for the `int?`. If they don't expect output, just put in 0.